src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs

```diff
@@ -1120,7 +1119,9 @@ public override BoundNode VisitConversion(BoundConversion conversion)
                 if (conversion.ConversionKind == ConversionKind.MethodGroup &&
                     conversion.SymbolOpt?.MethodKind == MethodKind.LocalFunction)
                 {
-                    return conversion;
+                    return new PartiallyLoweredLocalFunctionReference(
+                        conversion,
+                        new Dictionary<Symbol, CapturedSymbolReplacement>(proxies));
```

Comment:
Also, it looks like you only need the proxies for the frames, not the whole deal. 
Perhaps it makes sense to filter out proxies that are not in _framePointers while cloning? 

---
In reply to: [85404558](https://github.com/dotnet/roslyn/pull/14736#discussion_r85404558) [](ancestors = 85404558,85396261)